app/views/application_records/_combined_totals.haml

```diff
@@ -1,19 +1,14 @@
-%table#eeo_table
-  %thead
-    %tr
-      %th Females by ethnicity
-      %th Total Applicants
-  - @records[:female_ethnicities].each do |ethnicity, count|
-    %tr
-      %td= ethnicity
-      %td= count
+- @records[:combined_data].each do |key, value|
+  %table#eeo_table
+    %thead
+      %tr
+        %th= "#{key}s by Ethnicity"
```

Comment:
"Males" and "females" are OK, but this might end up looking not great if we incorporate more gender options.